code/script.js

```diff
@@ -225,101 +232,173 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret); //Writes out the randomly selected secret person in the console for test
 }
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
-}
+  //1. The Board is loaded when site is loaded/reloaded. 
+  generateBoard(); // shows the playing board
+  setSecret();// chose a random character and set as secret
+  selectQuestion(); //this will save the selected values into variables in the next function.
+
+;}
+
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
-
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+  // These variables stores what option group/category (hair, eyes, accessories, other) the question belongs to
+  // and the actual value of the question selected.
+  let category = questions.options[questions.selectedIndex].parentNode.label;
+  console.log(category);
+  let value = questions.options[questions.selectedIndex].value;
```

Comment:
Here you can try this alternative for shortening the code:
let value = category. value;